File: src/models/UserModel.ts
export class UserModel
{
    private id?: number;
    private cpf: string;
    private senha: string;

    constructor(cpf: string, senha: string, id?: number)
    {
        this.id = id;
        this.cpf = cpf;
        this.senha = senha;
    }

    public getId(): number | undefined
    {
        return this.id;
    }
    public setId(id: number): void
    {
        this.id = id;
    }

    public getCpf(): string
    {
        return this.cpf;
    }
    public setCpf(cpf: string): void
    {
        this.cpf = cpf
    }

    public getSenha(): string
    {
        return this.senha;
    }
    public setSenha(senha: string): void
    {
        this.senha = senha;
    }
}

File: BackEnd/src/main/java/SoftPet/backend/model/UserModel.java
package SoftPet.backend.model;

public class UserModel
{
    private Long id;
    private String cpf;
    private String senha;

    public UserModel(Long id, String cpf, String senha)
    {
        this.id = id;
        this.cpf = cpf;
        this.senha = senha;
    }

    public UserModel(String cpf, String senha)
    {
        this.cpf = cpf;
        this.senha = senha;
    }

    public UserModel()
    {
        this(0L, "", "");
    }

    public Long getId() {
        return id;
    }
    public void setId(Long id) {
        this.id = id;
    }

    public String getCpf() {
        return cpf;
    }
    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getSenha() {
        return senha;
    }
    public void setSenha(String senha) {
        this.senha = senha;
    }
}
